packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx

```diff
@@ -85,6 +90,20 @@ export const AccountsMenu = () => {
 
     const { discovery } = useDiscovery();
 
+    if (!device || !discovery) {
+        return null;
+    }
+
+    const failed = getFailedAccounts(discovery);
+
+    const list = sortByCoin(accounts.filter(a => a.deviceState === device.state).concat(failed));
+    const filteredAccounts =
+        searchString || coinFilter
```

Comment:
Mabye something like: 

`searchString.trim() !== ""`

Reply:
I think it is not really necessary

Reply:
I see that `accountSearchFn` does the trim itself 

ok